honeybee_radiance/modifierset.py

```diff
@@ -197,17 +219,21 @@ def modifiers(self):
             self.roof_ceiling_set.modifiers + \
             self.aperture_set.modifiers + \
             self.door_set.modifiers + \
-            self.shade_set.modifiers
+            self.shade_set.modifiers + \
+            [self.air_wall_modifier]
 
     @property
     def modified_modifiers(self):
         """List of all modifiers that are not defaulted within the set."""
-        return self.wall_set.modified_modifiers + \
+        modified_modifiers = self.wall_set.modified_modifiers + \
             self.floor_set.modified_modifiers + \
             self.roof_ceiling_set.modified_modifiers + \
             self.aperture_set.modified_modifiers + \
             self.door_set.modified_modifiers + \
             self.shade_set.modified_modifiers
+        if self._air_wall_modifier is not None:
```

Comment:
Like all of the other modifiers on the ModifierSet, when there's no specific modifier assigned for air walls, it defaults to the global modifier, which is a 100% transmittance surface. This option is mostly here so that people can change the air wall modifier if they want.